public/styles/main.css

```
@@ -0,0 +1,303 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap");
```

Stylelint (17.6.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap"" (import-notation)

(import-notation)